muffnn/autoencoder/autoencoder.py

```diff
@@ -0,0 +1,682 @@
+"""
+Autoencoder in scikit-learn style with TensorFlow
+"""
+import logging
+import re
+import warnings
+
+import numpy as np
+import tensorflow as tf
+import scipy.sparse as sp
+
+from sklearn.base import TransformerMixin, BaseEstimator
+from sklearn.utils import check_array, check_random_state
+from sklearn.utils.validation import NotFittedError
+
+from ..core import TFPicklingBase, affine
```

Comment:
hmm... i *think* this issue is orthogonal to how packages get loaded from the filesystem.  iirc, if you're in a directory with a subdirectory that's a package, that'll take priority regardless of relative or absolute importing.